include/env.bash

```diff
@@ -108,6 +108,7 @@ CB_SMARTSENSE_ID - SmartSense subscription ID
 CB_TEMPLATE_DEFAULTS - Comma separated list of the default templates what Cloudbreak initialize in database
 CB_UI_MAX_WAIT - Wait timeout for `cbd start-wait` command
 CB_LOCAL_DEV - If set true then Cloudbreak and Periscope containers will not be created
+CBD_LOCAL_DIRECTORY - The location of the cbd-local directory. Defaults to `pwd`.
```

Comment:
Does it make sense to make this configurable?

Reply:
cbd does not work for me on Linux without this because /home/mike/cbd-local maps to /hosthome/mike/cbd-local on the docker machine. I'm hesitant to mess with this mapping on the docker machine itself -) I assume it's this way for a reason.